app/src/main/java/app/grapheneos/camera/ui/activities/VideoCaptureActivity.kt

```diff
@@ -18,8 +20,10 @@ class VideoCaptureActivity : CaptureActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        whiteOptionCircle = findViewById(R.id.white_option_circle)
-        playPreview = findViewById(R.id.play_preview)
+        binding = ActivityMainBinding.inflate(layoutInflater)
```

Comment:
`VideoCaptureActivity` is a subclass of the `MainActivity`, `ActivityMainBinding.inflate` is called during `super.onCreate(savedInstanceState)`